specification/sql/resource-manager/Microsoft.Sql/2017-03-01-preview/managedDatabases.json

```diff
@@ -0,0 +1,556 @@
+          },
+          "202": {
+            "description": "Accepted"
+          },
+          "204": {
+            "description": "The specified managed database does not exist."
+          }
+        },
+        "x-ms-long-running-operation": true,
+        "x-ms-examples": {
+          "Delete managed database": {
+            "$ref": "./examples/ManagedDatabaseDelete.json"
+          }
+        }
+      },
+      "patch": {
+        "tags": [
+          "ManagedDatabases"
+        ],
+        "description": "Updates an existing database.",
+        "operationId": "ManagedDatabases_Update",
+        "parameters": [
+          {
+            "$ref": "#/parameters/ResourceGroupParameter"
+          },
+          {
+            "$ref": "#/parameters/ManagedInstanceNameParameter"
+          },
+          {
+            "$ref": "#/parameters/DatabaseNameParameter"
+          },
+          {
+            "name": "parameters",
+            "in": "body",
+            "description": "The requested database resource state.",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/ManagedDatabaseUpdate"
+            }
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Successfully updated the database.",
+            "schema": {
+              "$ref": "#/definitions/ManagedDatabase"
+            }
+          },
+          "default": {
+            "description": "*** Error Responses: ***\n\n * 404 SubscriptionDoesNotHaveServer - The requested server was not found\n\n * 404 ResourceNotFound - The requested resource was not found."
+          },
+          "202": {
+            "description": "Accepted"
+          }
+        },
+        "x-ms-long-running-operation": true,
+        "x-ms-examples": {
+          "Updates a managed database with minimal properties": {
+            "$ref": "./examples/ManagedDatabaseUpdateMin.json"
+          },
+          "Updates a managed database with maximal properties": {
+            "$ref": "./examples/ManagedDatabaseUpdateMax.json"
+          }
+        }
+      }
+    }
+  },
```

Comment:
list API to list all the managedDatabases under a managedInstance missing?

Reply:
Yes, my mistake.